docs/backends/causely.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: "Causely"
+---
+
+## Configuring the Causely Backend
+
+In order to add a new Causely backend, input the Causely Mediator endpoint URL into Odigos UI.
```

Comment:
Sometimes users get confused and are unsure if they need to add protocol `http://`, URL path, port etc. Since this endpoint is `grpc`, I guess the endpoint should be in the format `host:port` which can be nice to state here.
Is the `Mediator endpoint URL` something that users just need to copy and paste from somewhere in causely UI? or do they need to construct it themselves?